src/App.tsx

```diff
@@ -1,14 +1,37 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
+// eslint-disable-next-line import/no-cycle
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { Todo } from './types/Todo';
+import { getTodos } from './api';
+
+export interface Filter {
+  title: string,
+  status: string;
+}
+
+function getFilteredTodos(todos:Todo[], filter: Filter) {
+  return todos.filter(todo => todo.title.toLowerCase().includes(filter.title.toLowerCase())
+  && (filter.status === 'all' || (filter.status === 'active' && !todo.completed) || (filter.status === 'completed' && todo.completed)));
+}
```

Comment:
Please make this function with switch case, because this code looks confusing. Also you can move this function to separate utils folder